app/movie/[imdbid]/page.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import * as React from "react";
+import { MovieField } from "@/lib/MovieField";
+import {HiMiniHeart,HiOutlineHeart,HiMiniStar,HiOutlineStar } from "react-icons/hi2";
+import { MoviesContext } from "@/context/movieData.context";
+import movieData from "../../../lib/top_100_movies.json";
+
+interface PropType {
+  params: {
+    imdbid: string;
+  };
+}
+
+const MovieDetail: React.FC<PropType> = ({ params }) => {
+  const [isFavourited, setIsFavourited] = React.useState(false);
+  const {addFavMovie, removeFavMovie} = React.useContext(MoviesContext);
+  const movie = movieData.find((movie) => movie.imdbid === params.imdbid);
+  const {
+    title,
+    rating,
+    id,
+    year,
+    image,
+    description,
+    trailer,
+    genre,
+    director,
+    writers,
+    imdbid
+  } = movie as MovieField;
+
+  const handleAddFav = (_movie: MovieField) => {
+    addFavMovie(_movie);
+    setIsFavourited(true)
+  }
+  const handleRemoveFav = (_imdbid: string) => {
+    removeFavMovie(_imdbid);
+    setIsFavourited(false)
+  }
+  return( 
+    <div>
+      <h1 className="text-4xl">{title}</h1>
+      <div className="flex items-center gap-2">
+      <span className="text-sm">SAVED TO WISHLIST</span>
```

Comment:
bug/a11y: Avoid words in capslock because of A11Y. Use CSS `transform: uppercase` instead. [resource](https://www.timdunklee.com/notes/use-css-text-transform-for-uppercase-letters-a11y/).